types/jstree/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for jsTree v3.3.4
+// Type definitions for jsTree v3.3.8
```

Comment:
Note that the expected format is `major.minor` since the patch version is not consumed.  The non-optional properties added below might break consumers of the 3.3.4 type who are not providing them.

Reply:
Hi,

I overlooked that - I changed my commit to contain only major.minor
it should be ready after checks

Reply:
Thanks!  Could you possibly enable the `dt-header` and `npm-naming` lint rules?  They're important for the functioning of our infrastructure.

To save you a step, you can clear up the warnings from those two rules by changing the first two lines to

```ts
// Type definitions for jstree 3.3
// Project: http://jstree.com/
```